programs/marginfi/src/instructions/marginfi_account/borrow.rs

```diff
@@ -103,7 +141,8 @@ pub fn lending_account_borrow<'info>(
             },
             bank: bank_loader.key(),
             mint: bank.mint,
-            amount: amount_pre_fee,
+            amount_pre_fee,
```

Comment:
Also, I'm not sure what is better UX, to receive the `amount` and borrow `amount + ori_fee` or to receive `amount` and borrow. I'd want to be consistent with other venues or align more with user demands.

Reply:
No preference on my end, I don't think there's a standard here.